lib/server/swarm.js

```diff
@@ -98,15 +108,19 @@ Swarm.prototype._onAnnounceUpdate = function (params, peer) {
     this.complete += 1
     this.incomplete -= 1
     peer.complete = true
+    var id = params.type === 'ws' ? params.peer_id : params.addr
+    this.peers.set(id, peer)
```

Comment:
I don't understand what you mean by changing this to `get()`, I am pushing the other changes but what do you mean?